www/js/services.js

```diff
@@ -16,10 +16,16 @@ angular.module('pvta.services', ['ngResource'])
     return $resource('http://bustracker.pvta.com/infopoint/rest/stops/get/:stopId');
 })
 
-.factory('Stops', function($resource){
+.factory('Stops', function($resource) {
     return $resource('http://bustracker.pvta.com/infopoint/rest/stops/getallstops');
 })
 
+.factory('NearestStops', function($resource){
+    return $resource('http://bustracker.pvta.com/infopoint/rest/Stops/Nearest?latitude=:latitude&longitude=:longitude', {latitude: "@latitude", longitude: "@longitude"}, {
+        query: {method: 'get', isArray: true}          
```

Comment:
This line is somewhat redundant, because the [documentation](https://docs.angularjs.org/api/ngResource/service/$resource) says that the query() method sets the HTTP method to **get** and the **isArray** property to true internally when called.